themes/default/content/blog/esc-kubernetes-cluster-and-app/index.md

```diff
@@ -0,0 +1,317 @@
+---
+title: "Create a Kubernetes Cluster and App with Pulumi ESC"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2023-11-17T14:05:53-07:00
+
+# The draft setting determines whether a post is published. Set it to true if
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: Use Pulumi ESC to manage configuration for a Kubernetes cluster.
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - levi-blackstone
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - esc
+    - kubernetes
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+
+Pulumi gives you great flexibility to [factor your infrastructure into reusable parts](/docs/using-pulumi/organizing-projects-stacks).
+One of the challenges in factoring code is managing the shared configuration that needs to pass between the subcomponents. [Pulumi ESC](/product/esc/)
+simplifies this problem by providing a common place for Pulumi Infrastructure as Code (IaC) programs and other cloud tools
+to share configuration and secrets. In this post, we will create a Kubernetes cluster, deploy an application to the created cluster,
+and then use `kubectl` to check on the deployed application.
+
+## Create a Kubernetes Cluster
+
+First, we create a Kubernetes cluster using the [pulumi-eks](/registry/packages/eks) component.
+
+```yaml
```

Comment:
I know it's more work, but can we use `pulumi convert` to easily create this example in other languages?